src/openMVG/features/svgVisualization.cpp

```diff
@@ -24,11 +51,11 @@ float getStrokeEstimate(const std::pair<size_t,size_t> & imgSize)
 
 void saveMatches2SVG(const std::string &imagePathLeft,
                      const std::pair<size_t,size_t> & imageSizeLeft,
-                     const std::vector<features::PointFeature> &keypointsLeft,
+                     const features::MapRegionsPerDesc &keypointsLeft,
```

Comment:
I think it will be more easier to add this function when we will have the use case